modules/library/src/main/scala/zio/elasticsearch/ElasticRequest.scala

```diff
@@ -717,7 +718,7 @@ object ElasticRequest {
     aggregation: ElasticAggregation,
     excluded: Chunk[String],
     included: Chunk[String],
-    index: IndexName,
+    index: String,
```

Comment:
I'm thinking about renaming this to "selector" or "selectors" and using `Chunk[String]` instead. Also, I would probably use the same name and type for all underlying private classes in the ElasticRequest.

@arnoldlacko @dbulaja98

Reply:
Maybe we should consider using both "selector" and "selectors" for places we have one index, and places where we have more than one.